File: projectUI/projectUI/Item.swift
//
//  Item.swift
//  projectUI
//
//  Created by Sarah Sherman on 1/26/16.
//  Copyright © 2016 Sarah Sherman. All rights reserved.
//

import Foundation



class Item {
    var iid: String
    var userName: String
    var location: String
    var itemName: String
    var detail: String
    var locationY: Double
    var locationX: Double
    var imageUrl: String
    var createdAt: String
    var founds: [NSDictionary]
    
    init(iid: String, userName: String, itemName: String, locationX: Double, locationY: Double, details: String, dateListed: String, imageUrl: String, location: String) {
        self.iid = iid
        self.userName = userName
        self.itemName = itemName
        self.location = location
        self.locationX = locationX
        self.locationY = locationY
        self.imageUrl = imageUrl
        self.detail = details
        self.createdAt = dateListed
        self.founds = [NSDictionary]()
    }}

File: foundItems/app/models/item.js
var mongoose = require('mongoose');
var itemSchema = new mongoose.Schema({
	userName: String,
	location: String,
	itemName: String,
	detail: String,
	locationY: Number,
	locationX: Number,
	imageUrl: String,
	createdAt: Date,
	founds: {type: Array}
});

module.exports = mongoose.model('Item', itemSchema);